manifests/.gitignore

```diff
@@ -0,0 +1 @@
+/custom manifests/
```

Comment:
not a thing, just don't add that file instead of adding mysterious gitignores :-)